client/containers/DashboardSubmissions/NewSubmissionWorkflowEditor.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Button } from '@blueprintjs/core';
+
+import { useLocalStorage } from 'client/utils/useLocalStorage';
+import { getEmptyDoc } from 'client/components/Editor';
+import { usePageContext } from 'utils/hooks';
+
+import { EditableSubmissionWorkflow } from './types';
+import SubmissionWorkflowEditor from './SubmissionWorkflowEditor';
+import StartWorkflowCallout from './StartWorkflowCallout';
+
+const createEmptyWorkflow = (): EditableSubmissionWorkflow => {
+	return {
+		instructionsText: getEmptyDoc(),
+		emailText: getEmptyDoc(),
+		targetEmailAddress: '',
+		enabled: false,
+		bannerContent: {
+			title: '',
+			body: getEmptyDoc(),
+		},
+	};
+};
+
+type Props = {
+	onWorkflowCreated: (w: EditableSubmissionWorkflow) => unknown;
```

Comment:
Eventually this component will be responsible for POSTing `/api/submissionWorkflows`, and it'll call this with the whole model (ID and all)